src/connector/src/parser/mod.rs

```diff
@@ -488,6 +488,7 @@ impl SourceStreamChunkRowWriter<'_> {
                             .map(|ele| ScalarRefImpl::Utf8(ele.split_id)),
                     ));
                 }
+                (_, &Some(AdditionalColumnType::Payload(_))) => parse_field(desc),
```

Comment:
Better to leave a comment showing that its actual handling resides in `KvEvent::access_field`.